test/utils/fixture/fixture.go

```diff
@@ -203,3 +211,11 @@ func getTeardownTime() time.Duration {
 	}
 	return val
 }
+
+func GetCoolDownTime() time.Duration {
+	return cooldownTime
+}
+
+func GetTeardownTime() time.Duration {
+	return teardownTime
+}
```

Comment:
I'm not sure we need these getters.

Reply:
right, we don't need these two. thanks for pointing this out.